### PR TITLE
T1028 "Windows Remote Management": split in several techniques

### LINKED ISSUE #1042: T1028 "Windows Remote Management": split in several techniques
T1028 currently has 5 atomic tests implemented:
1. Atomic Test #1 - Enable Windows Remote Management
2. Atomic Test #2 - PowerShell Lateral Movement
3. Atomic Test #3 - WMIC Process Call Create
4. Atomic Test #4 - Psexec
5. Atomic Test #5 - Invoke-Command

T1028 is very specific to "Windows Remote Management", also called WinRM which is used by PowerShell Remoting (and others) but not by psexec, mmc20 or remote WMI.

Therefore, I suggest keeping only those in T1028:
* Atomic Test #1 - Enable Windows Remote Management
* Atomic Test #5 - Invoke-Command

And here are my suggestions for the others:
* Atomic Test #2 - PowerShell Lateral Movement: it uses PowerShell but actually the technique is based on "mmc20" which has its own T1175 "Component Object Model and Distributed COM" currently non existent
* Atomic Test #3 - WMIC Process Call Create: I'd move it to T1047 - Windows Management Instrumentation which already has a very similar test "Atomic Test #6 - WMI Execute Remote Process"
* Atomic Test #4 - Psexec: it's actually T1035 - Service execution, which already has a relevant test "Atomic Test #2 - Use PsExec to execute a command on a remote host"

What do you think? Would you like me to implement these suggestions?

### PR DESCRIPTION
**Pull request by cnotin**
**Details:**
I deleted and copied some code as explained in #1042
I've added username and password for psexec that were present in deleted code

**Testing:**
I did not test it!
I didn't re-gen the GUIDs

**Associated Issues:**
Fixes #1042

### CODE REVIEW

File: atomics/T1218.003/T1218.003.yaml [hunk 1]
```diff
@@ -49,4 +49,24 @@ atomic_tests:
     command: |
       cmstp.exe /s #{inf_file_uac} /au
     name: command_prompt
+- name: PowerShell Lateral Movement
```
Comment (mgraeber-rc) [inline atomics/T1218.003/T1218.003.yaml:52]: Did you mean to add this to [T1218.003 - CMSTP](https://attack.mitre.org/techniques/T1218/003/)? I see you also moved it to T1021.003 which would be the correct sub-technique.

Comment (cnotin) [reply atomics/T1218.003/T1218.003.yaml]: Fixed!

**Comment by cnotin** [comment]
Fixed, indeed I pushed a mistake I didn't mean to!
I suggest squashing the PR :)

**Review by mgraeber-rc — APPROVED** [review]

**Comment by mgraeber-rc** [comment]
Thank you!

**Pull request merged**